how-to-look-premium/readme.md

```diff
@@ -0,0 +1,24 @@
+# How to look premium
+
+<img src="http://i.imgur.com/4ETDkim.jpg" widht="100%">
+
+So, you have set up an awesome service and you are ready to take it to the market. You have done your research and you know you have more to offer than much of the competition. Therefore you decide to put your pricing in a higher segment. Yes, you are a premium service, spending much time and effort on your product, and your clients will have to pay for that. But how to convince them you are worth the money? Here's a few points to help you get started, inspired by [this article](https://www.entrepreneur.com/article/217176).
```

Comment:
I would mention the source as well if you got a lot from them. So maybe inspired by [this article] from entrepreneur.com.